app/views/storage/_storage_pod_folders.html.haml

```diff
@@ -5,7 +5,7 @@
       %tbody
         - @folders.each do |f|
           %tr{:title => _("View '%{name}' Cluster") % {:name => f.name},
-            :onclick => "miqTreeActivateNode('storage_pod_tree', 'dsc_xx-#{to_cid(f.id)}');"}
+            :onclick => "miqTreeActivateNode('storage_pod_tree', 'xx-#{to_cid(f.id)}');"}
```

Comment:
@mzazrivec Actually, `miqTreeActivateNode` needs the key exactly as the tree generated it, which probably does include `to_cid`.

Either way, seems like the bug is probably present since ManageIQ/manageiq#8004, adding `euwe/yes`.